jekyll/_cci2/server-3-whats-new.adoc

```diff
@@ -20,6 +20,40 @@ patch releases and quarterly feature releases.
 
 toc::[]
 
+== Release 3.4.0
+
+=== Upgrade notes
+
+Before upgrading to v3.4.0, delete the deployment named `circleci-server-kube-state-metrics`:
+
+* Run `kubectl delete deployment/circleci-server-kube-state-metrics --cascade=orphan`
+* Redeploy the app from the KOTS admin console
```

Comment:
```suggestion
* Deploy the app from the KOTS admin console
```

If the user deletes the deployment before hitting Deploy then there's no need to hit Redeploy

Reply:
Thanks!